### ISSUE TITLE
css fixed가 먹히지 않습니다..이틀을 이것 때문에 막히네요ㅠㅠ 도와주세요

### ISSUE THREAD

**Issue by lee-insu** [codingeverybody/codingyahac#943]
# 해결하고자 하는 문제
하단고정을 하고 싶어서 
css foot부분에 position을 fixed로 설정 했는데 움직임

# 코드 혹은 오류
<!DOCTYPE html>
<html lang="ko">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>쉬잇-가볍게 일상 탈출</title>
    <style>
@media screen and (max-width:480px) {
    body{
        width: 480px;
        height: 812px;
        background-color: #fbfbfb;
        margin:0px;
      }
.header {
    width: 480px;
    height: 51px;
    position: relative; top:0px; left:0px;
    background-color: #ffffff;
  }
.container {
  z-index: 1;
}
.header-title {
    width: 100%;
    height: 31px;
    position: relative; top:10px; left:30.8% ;
    font-family: Jalnan;
    font-size: 31px;
    font-weight: normal;
    font-stretch: normal;
    font-style: normal;
    line-height: 1.26;
    letter-spacing: normal;
    color: #ff6868;
  }
  .header-sub-title{
    display:none
  }
.search{
    width: 277px;
    height: 33px;
   position: absolute; top:59px; left: 40px;
    border: solid 2px #ff6868;
    background-color: #ffffff;
  }
.search-btn{
    border:none;
    position: absolute; top:61px; left:326px;
    background-color:#fbfbfb;
}
.grid1{
  position: relative; top:50px;
    width: 100%;
    height: 317px;
    background-color: #ffffff;
  }
 
.grid1-text{
    width: 238px;
    height: 21px;
    position: relative; top: 14px; left:62px;
    font-family: 빙그레 메로나체;
    font-size: 16px;
    font-weight: normal;
    font-stretch: normal;
    font-style: normal;
    line-height: 1.31;
    letter-spacing: normal;
    text-align: left;
    color: #535353;
  }
.grid2{
    width: 100%;
    height: 317px;
    position: relative; top:65px;
    background-color: #ffffff;
    
  }
.grid2-text{
    width: 205px;
    height: 21px;
    font-family: 빙그레 메로나체;
    position: relative; top:10px; left:80px;
    font-size: 16px;
    font-weight: normal;
    font-stretch: normal;
    font-style: normal;
    line-height: 1.31;
    letter-spacing: normal;
    text-align: left;
    color: #535353;

  }
#foot {
  width: 164px;
  height: 23px;
  position: fixed; bottom: 42px;
  font-family: 빙그레 메로나체;
  font-size: 18px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 1.28;
  letter-spacing: normal;
  text-align: center;
  color: #ffffff;
  background-color: #ff6868;
}

 }
</style>
</head>

<body>
    <header class="header">
        <div class="header-title">
            쉬잇
        </div>
        <div class="header-sub-title">
            가볍게 일상을 탈출해요
        </div>

        <div class="header-search">
            <form method="POST" role="search" onsubmit="">
                <input type="text" class="search">
            </form>
            <button type="button" class="search-btn" type="submit">
                <img src="img/search.png" srcset="img/search2x.png 2x,
                            img/search3x.png 3x" class="search-btn-img">
            </button>
        </div>
    </header>
    <div class="container">
        <div class="grid1">
            <div class="grid1-text">
                당신을 위한 거짓말을 준비해봤어요.
            </div>
            <section class="grid1-section">
                <ul>
                    <li></li>
                    <li></li>
                </ul>
            </section>
        </div>
        <div class="grid2">
            <div class="grid2-text">
                여러분의 거짓말을 보여주세요.
            </div>
            <div class="grid2-section">

            </div>
        </div>
    </div>
    <div id="foot">
        나의 필살 거짓말 쓰기
    </div>
</body>

</html>

# 환경
크롬 최신버전

**Comment by smartbos** [comment]
`@media screen and (max-width:480px) {}`
이 구문 때문에 작성하신 CSS가 가로 480px 미만일 때만 동작하고 있습니다.

**Comment by lee-insu** [comment]
네 감사합니다. 제가 크롬 개발자도구를 이용해서 iphone x 사이즈로 작업해서 확인하는데 작동이 안되더라구요.. 그래서 혹시나해서 미디어쿼리를 비활성화 시켰는데도 작동을 안 해서.. 코드에 다른 오류가 더 있을까요?
css 에서 container,  header, header-title을 비활성화 시키면 작동을 하는데 왜 그런지 이유를 모르겠습니다 ㅠㅠ

**Comment by studroid** [comment]
위 현상은 크롬 개발자 도구의 버그 같습니다 :)
Responsive 옵션으로 보시는 경우, fixed 옵션이 제대로 반영되지 않는 현상이 있네요.
max-width를 늘려서 개발자 도구 없이 볼 수 있는 정도로 작업을 해보시면 정상적으로 동작하는 것을 확인할 수 있으실 듯합니다.

혹시 다른 상태라면 알려주세요!
